src/Analyzers/CSharp/Analyzers/RemoveUnnecessaryDiscardDesignation/CSharpRemoveUnnecessaryDiscardDesignationDiagnosticAnalyzer.cs

```diff
@@ -19,6 +19,7 @@ internal sealed class CSharpRemoveUnnecessaryDiscardDesignationDiagnosticAnalyze
     {
         public CSharpRemoveUnnecessaryDiscardDesignationDiagnosticAnalyzer()
             : base(IDEDiagnosticIds.RemoveUnnecessaryDiscardDesignationDiagnosticId,
+                   EnforceOnBuildValues.RemoveUnnecessaryDiscardDesignation,
```

Comment:
❔ Would it be cleaner to have the base class look up this value from the diagnostic ID already getting provided?

Reply:
Given that new analyzers would have to anyways define a new constant in `EnforceOnBuildValues`, I think this feels much more cleaner and explicit